discovery-infra/start_discovery.py

```diff
@@ -370,7 +385,13 @@ def nodes_flow(client, cluster_name, cluster, image_path):
 
     tf_folder = utils.get_tf_folder(cluster_name, args.namespace)
     utils.recreate_folder(tf_folder)
-    copy_tree(consts.TF_TEMPLATE, tf_folder)
+
+    is_none_platform = args.sec_master_count or args.sec_worker_count
```

Comment:
As i said, i think better to have flag that says "none_platform" and not to looks on those flags. I think don't need them at all